src/lib/y2partitioner/widgets/disk_table.rb

```diff
@@ -69,22 +71,16 @@ def header
         ]
       end
 
+      HELP_FIELDS = [:device, :size, :format, :encrypted, :type, :fs_type,
+                     :label, :mount_point, :start_cyl, :end_cyl].freeze
```

Comment:
This is error prone, just like defining the table columns elsewhere. We should eventually come up with something that is less likely to break if you need to add/move/remove a table column.